calico_node/Makefile

```diff
@@ -1,6 +1,6 @@
 ###############################################################################
 # Versions:
-RELEASE_STREAM?=v2.3
+RELEASE_STREAM?=master
```

Comment:
Might not be needed after your change but does it make sense for `make st` to run with `v2.3` versions?  I'd kinda expect that to use the current code/master.